openquake/commands/prepare_site_model.py

```diff
@@ -29,12 +29,48 @@
 from openquake.hazardlib.geo.utils import assoc
 from openquake.risklib.asset import Exposure
 from openquake.commonlib.writers import write_csv
-from openquake.commonlib.util import compose_arrays
-
 
 F32 = numpy.float32
 SQRT2 = 1.414
 vs30_dt = numpy.dtype([('lon', F32), ('lat', F32), ('vs30', F32)])
+req_site_params = {'vs30', 'z1pt0', 'z2pt5'}
+
+
+def calculate_z1pt0(vs30):
+    '''
+    Reads an array of vs30 values (in m/s) and
+    returns the depth to the 1.0 km/s velocity horizon (in m)
+
+    :param vs30: the shear wave velocity (in m/s) at a depth of 30m
+    '''
```

Comment:
Is there a literature source for this conversion model between Vs30 and Z1.0. It looks like the Abrahamson & Silva (2008) model, so this should be clarified in the doc string.

Reply:
Changed to use the Chiou & Youngs (2014)